ACPs/30-avalanche-warp-x-evm.md

```diff
@@ -0,0 +1,216 @@
+```text
+ACP: 30
+Title: Integrate Avalanche Warp Messaging into the EVM
+Author(s): Aaron Buchwald <aaron.buchwald56@gmail.com>
+Status: Proposed
```

Comment:
Skipping the discussion section here since activating AWM in the EVM has already been generally established as one of the most important new features on Avalanche and has already been under development since before the ACP process.

Happy to create one if there's an interest or preference

Reply:
Ah, I think that discussion section is for the phase 2 discussion not the previous "ideas" discussion. You should leave it.

Reply:
Ah I misinterpreted the instructions since creating a discussion comes before opening a PR: https://github.com/avalanche-foundation/ACPs/#step-1-post-your-idea-to-github-discussions. I'll add this back to fill the link in later.